packages/react-sdk/src/components/html.tsx

```diff
@@ -0,0 +1,19 @@
+import { forwardRef, type ElementRef } from "react";
+
+type Props = {
+  code?: string;
+};
+
+export const Html = forwardRef<ElementRef<"div">, Props>((props, ref) => {
+  const { code, ...rest } = props;
+  return (
+    <div
+      {...rest}
+      ref={ref}
+      style={{ display: code?.trim().length !== 0 ? "contents" : "block" }}
+      dangerouslySetInnerHTML={{ __html: code ?? "" }}
```

Comment:
didn't we want to use insertAdjacentHTML or is react using it?

Reply:
Just first draft